src/commands/kubeval.yaml

```diff
@@ -8,7 +8,7 @@ steps:
       name: kubeval
       command: |
         if [ -d "helm/<<parameters.chart>>/ci" ]; then
-          for t in $(ls helm/<<parameters.chart>>/ci/*.yaml); do helm template --values $t helm/<<parameters.chart>> | kubeval --ignore-missing-schemas - ; done
+          for t in $(ls helm/<<parameters.chart>>/ci/*.yaml); do helm template --values $t helm/<<parameters.chart>> | kubeval --ignore-missing-schemas --kubernetes-version skip - ; done
```

Comment:
question: without this change, would `kubeval` do the same as `deprek8ion`? What I'm asking is, could we remove `deprek8ion` and use only `kubeval` or are they different things?

Reply:
Yes, without this change, would `kubeval` do the same as `deprek8ion`. We could in theory remove `deprek8ion` and use only `kubeval`.

I introduce `kubeval` because it can detect overlapping k8s resources as well.

Reply:
Cool. No blocking here, but we should definitely take a look and see if we can get rid of `deprek8ion` and make things simpler.

Reply:
How about we leave this here, see if we are happy with `kubeval` schema validation and drop `deprek8ion` later on then ?